sycl/doc/extensions/Avoid_event_creation/avoid_event_creation.asciidoc

```diff
@@ -0,0 +1,144 @@
+= Avoid event creation
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+// This is necessary for asciidoc, but not for asciidoctor
+:cpp: C++
+
+== Notice
+
+IMPORTANT: This specification is a draft.
+
+Copyright (c) 2021 Intel Corporation. All rights reserved.
+
+NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are
+trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc.
+used by permission by Khronos.
+
+NOTE: This document is better viewed when rendered as html with asciidoctor.
+GitHub does not render image icons.

```

Comment:
You can remove this; this was true before, but we've started formatting extensions for GitHub instead.

Reply:
You should also add a section about a feature test macro.  See https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/Uniform/Uniform.asciidoc for an example.

Reply:
ok, done.